x-pack/plugins/apm/public/tutorial/config_agent/get_policy_options.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import { APIResponseType } from './';
+
+const POLICY_ELASTIC_AGENT_ON_CLOUD = 'policy-elastic-agent-on-cloud';
```

Comment:
This is also defined in the other branch at `x-pack/plugins/apm/server/lib/fleet/get_cloud_apm_package_policy.ts` (https://github.com/elastic/kibana/pull/102682/files#diff-241c486c7460099bbef65a8edb7db5b8fe7a0218aa0ebb177e18e51ec0b3c744R12)

Reply:
When you merge your PR I'll change it